src/index.html

```diff
@@ -22,6 +22,402 @@
+          alt="APPLE A1419 iMac 27'"
+        />
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__description">Product code: 195434</p>
+
+        <div class="card__wrapper-stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__wrapper-price">
+          <p class="card__price">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27'"
+        />
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__description">Product code: 195434</p>
+
+        <div class="card__wrapper-stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__wrapper-price">
+          <p class="card__price">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27'"
+        />
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__description">Product code: 195434</p>
+
+        <div class="card__wrapper-stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__wrapper-price">
+          <p class="card__price">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+        >
+          Buy
+        </a>
```

Comment:
The repeated `card` sections should be generated using a loop in SCSS for better maintainability. Also, since each card is a BEM block, its styles should be in a separate SCSS file and imported into the main SCSS file.